### PR TITLE
INFRA: Fix PPID cmd

### PR DESCRIPTION
**Pull request by Alexey-Rivkin**
## What
Fix cmd to find the parent process ID (PPID) for zombie processes.
Also improved zombie search. 

## Why
Wrong PPID was the reason that the script couldn't kill some zombies.

### CODE REVIEW

**Comment by Alexey-Rivkin** [comment]
Per Yossi's advice, only kill zombies with high (above 85%) CPU consumption.

File: buildlib/tools/zombie_killer.sh [hunk 1]
```diff
@@ -1,20 +1,20 @@
-#!/bin/bash -eEx
+#!/bin/bash -eE
 
-# Search for zombie or defunct processes; kill them all
-out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+# Search for zombie with high CPU usage; kill them all
+out=$(ps -eo pid,stat,%cpu | grep Z | awk '$3>85.0 {print $1}')
```
Comment (artemry-nv) [inline buildlib/tools/zombie_killer.sh:4]: Why don't we terminate all the zombie processes?

File: buildlib/tools/zombie_killer.sh [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-# Search again; tell parents you killed their kids; kill parents
-out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+# Search for survivors; tell parents you killed their kids; kill parents
+out=$(ps -eo pid,stat,%cpu | grep Z | awk '$3>85.0 {print $1}')
 for i in $out ; do
-  pid_parent=$(ps -o ppid=$i)
+  pid_parent=$(ps -p "$i" -o ppid=)
   kill -s SIGCHLD $pid_parent
   kill -9 $pid_parent
```
Comment (artemry-nv) [inline buildlib/tools/zombie_killer.sh:19]: Why do we need to terminate parent process with SIGKILL? As you mentioned parent process may be a docker process

File: buildlib/tools/zombie_killer.sh (hunk 1 above)
Comment (Alexey-Rivkin) [reply buildlib/tools/zombie_killer.sh:4]: Most zombies are harmless, but we may kill them just out of spite.

File: buildlib/tools/zombie_killer.sh (hunk 2 above)
Comment (Alexey-Rivkin) [reply buildlib/tools/zombie_killer.sh:19]: If a zombie couldn't be dealt with directly, approaching its parents is the only choice left.
For the high CPU-eating zombies it well worth the hustle.

**Review by yosefe — APPROVED** [review]

File: buildlib/tools/zombie_killer.sh (hunk 2 above)
Comment (artemry-nv) [reply buildlib/tools/zombie_killer.sh:19]: But if we parent for example `init` or another important process?

File: buildlib/tools/zombie_killer.sh (hunk 1 above)
Comment (artemry-nv) [reply buildlib/tools/zombie_killer.sh]: They may occupy resources and slow down the system. If we don't terminate all the zombie processes maybe at least print all the remaining ones in the end (for future debug purposes)?

Comment (Alexey-Rivkin) [reply buildlib/tools/zombie_killer.sh]: We do kill them all now, changed per your recommendation.

File: buildlib/tools/zombie_killer.sh (hunk 2 above)
Comment (artemry-nv) [reply buildlib/tools/zombie_killer.sh:19]: It appears we cannot terminate init even with SIGKILL.

**Pull request merged**